Manifest.toml

```diff
@@ -0,0 +1,38 @@
+[[Base64]]
```

Comment:
I don't think we need to check in the Manifest. Libraries usually don't.

Reply:
I had a long discussion about this with Stefan.  You should read the thread in #pkg-usage.  I'll link you on Slack.